test/utils/urlMatcher.test.ts

```diff
@@ -98,7 +98,7 @@ test('getPullRequestAPIURL', async () => {
   expect(await getPullRequestAPIURL('https://github.com/open-sauced/ai/pull/164')).toBe('https://api.github.com/repos/open-sauced/ai/pulls/164')
   expect(await getPullRequestAPIURL('https://github.com/open-sauced/ai/pull/163')).toBe('https://api.github.com/repos/open-sauced/ai/pulls/163')
   expect(await getPullRequestAPIURL('https://github.com/open-sauced/ai/compare/some-branch')).toBe('https://api.github.com/repos/open-sauced/ai/compare/beta...some-branch')
-  expect(await getPullRequestAPIURL('https://github.com/tailwindlabs/tailwindcss/compare/some-branch')).toBe('https://api.github.com/repos/tailwindlabs/tailwindcss/compare/master...some-branch')
+  expect(await getPullRequestAPIURL('https://github.com/tailwindlabs/tailwindcss/compare/some-branch')).toBe('https://api.github.com/repos/tailwindlabs/tailwindcss/compare/next...some-branch')
```

Comment:
This fixes the failure as it looks like the `tailwindlabs/tailwindcss` project change a branch name, but, and not for this PR, but I'd probably mock the network request that gets the default branch name for all these tests, i.e.

https://github.com/open-sauced/ai/blob/7ab9847ee33350709e70facf5465fcd6c90c63d9/src/utils/urlMatchers.ts#L73-L74